api/diagnostics_test.go

```diff
@@ -169,10 +168,12 @@ func TestGetAllStatus(t *testing.T) {
 
 func TestIsSnapshotAvailable(t *testing.T) {
 	tools := &fakeDCOSTools{}
+	cfg := testCfg()
+	defer os.Remove(cfg.FlagDiagnosticsBundleDir)
```

Comment:
Is it possible that this directory contains files? If so `RemoveAll` would be more robust.